_posts/2021-05-10-helm-charts-for-wildfly.adoc

```diff
@@ -0,0 +1,226 @@
+* deploy an application image with all the resources and configuration needed to integrate it on OpenShift.
+
+By default, the two steps are connected: the `build` step's output is an application image that becomes the input of the `deploy` step.
+However it is possible to enable or disable these steps: if you have another way to build your application image, you can disable the `build` step and still deploys it by specifying where your application image can be pulled.
+ 
+All the chart configuration is described in the https://github.com/wildfly/wildfly-charts/blob/main/charts/wildfly/README.md[Chart documentation].
+In this post, we will focus on a simple MicroProfile application to show you how the Helm chart simplifies its deployment on OpenShift.
+
+## MicroProfile Application Example
+
+We will use one of WildFly quickstart as an example: https://github.com/wildfly/quickstart/tree/master/microprofile-config[microprofile-config].
+We will not cover the code of this example but the important things to mention are:
+
+* This application will be running as a https://docs.wildfly.org/bootablejar/[WildFly Bootable Jar]
+* It uses MicroProfile API and provides a HTTP endpoint. It provision the `jaxrs-server` and `microprofile-platform` layers so that the running WildFly server contains all the bits (and only those!) to run these features.
+* Its HTTP endpoint uses MicroProfile Config to read a config property from the `CONFIG_PROP` environment variable and returns its value in the HTTP response.
+
+## Install Helm and have access to an OpenShift cluster.
+
+Please refer to https://helm.sh/docs/intro/install/[Installing Helm] page to install Helm in your environment.
+
+We will use https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] as our OpenShift cluster and make sure that we are logged in the cluster and that the `oc` command applies to that cluster.
+
+## Install Helm Repository for WildFly Chart
+
+The `wildfly` Chart can be installed from the https://docs.wildfly.org/wildfly-charts/ repository
+
+[source,nowrap]
+----
+$ helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+"wildfly" has been added to your repositories
+$ helm search repo wildfly
+NAME                    CHART VERSION   APP VERSION     DESCRIPTION
+wildfly/wildfly         1.3.0           23.0            Build and Deploy WildFly applications on OpenShift$ helm search repo wildfly
+----
+
+The first thing to notice is that the `APP VERSION` points to `23.0`. By default, the Chart will deploy your application using the latest WildFly 23.0 release (23.0.2.Final at the time of this writing).
+
+## Install the Application on OpenShift
+
+Once the `wildfly` chart is added, we can instal a Helm *release* by using the `helm install` command with the name of the release as well as a YAML configuration file that contains all the settings to build and deploy the application:
```

Comment:
I believe:

s/`wildfly` chart/`wildfly` Chart

Just because you have been using before Chart in uppercase